leveldb/leveldb.go

```
@@ -0,0 +1,159 @@
+	data, err := s.db.Get(key, nil)
+	if err != nil {
+		return nil, nil
+	}
+
+	var stored item
+	if err := json.Unmarshal(data, &stored); err != nil {
+		return data, nil
+	}
+
+	if !stored.ExpireAt.IsZero() && time.Now().After(stored.ExpireAt) {
+		s.db.Delete(key, nil)
```

golangci-lint (1.62.2)

Error return value of `s.db.Delete` is not checked

(errcheck)